config/jobs/kubernetes/sig-scalability/sig-scalability-periodic-jobs.yaml

```diff
@@ -1017,3 +1017,72 @@ periodics:
         limits:
           cpu: 2
           memory: 6Gi
+
+- interval: 2h
+  cluster: k8s-infra-prow-build
+  name: ci-kubernetes-benchmark-list
+  labels:
+    preset-service-account: "true"
+    preset-k8s-ssh: "true"
+    preset-e2e-scalability-common: "true"
+    preset-e2e-scalability-periodics: "true"
+    preset-e2e-scalability-periodics-master: "true"
+  decorate: true
+  decoration_config:
+    timeout: 90m
+  extra_refs:
+  - org: kubernetes
+    repo: kubernetes
+    base_ref: master
+    path_alias: k8s.io/kubernetes
+  - org: kubernetes
+    repo: perf-tests
+    base_ref: master
+    path_alias: k8s.io/perf-tests
+  annotations:
+    testgrid-dashboards: sig-scalability-benchmarks
+    testgrid-tab-name: gce-benchmark-list
+  spec:
+    containers:
+    - image: gcr.io/k8s-staging-test-infra/kubekins-e2e:v20250121-4aed057712-master
+      command:
+      - runner.sh
+      - /workspace/scenarios/kubernetes_e2e.py
+      args:
+      - --cluster=gce-scale-cluster
+      - --extract=ci/latest
+      - --gcp-master-size=n2-standard-32
+      - --gcp-node-size=e2-standard-32
+      - --gcp-node-image=gci
+      - --gcp-nodes=1
+      - --gcp-project-type=scalability-scale-project
```

Comment:
be careful approving jobs with `--gcp-project-type=scalability-scale-project`, this is the 5k node type which we only have one of.

This should be `scalability-project`, I think https://github.com/kubernetes/test-infra/pull/34305